pyproject.toml

```diff
@@ -56,7 +56,6 @@ dev = [
   "arize[AutoEmbeddings, LLM_Evaluation]",
 ]
 experimental = [
-  "openai",
```

Comment:
This is needed in `classify_relevance`. It will break some of our tutorials to remove it.

Reply:
It probably does make sense to remove `openai` as a dependency, but we would need to dynamically import `openai` inside of `classify_relevance` [here](https://github.com/Arize-ai/phoenix/blob/919bf202ea1b41a104a4b2f8d8ad364d03e1c0b2/src/phoenix/experimental/evals/retrievals.py#L65). We'd also need to make sure `openai` is explicitly installed in all of the notebooks where `classify_relevance` is used.

Reply:
I'd leave that to a future PR given that we need this PR asap